### PR TITLE
Implement getting kernel module version

### PR DESCRIPTION
**Pull request by mkpankov**
Handle second part of point 2 of https://github.com/whamcloud/integrated-manager-for-lustre/issues/1187#issuecomment-559830336

Signed-off-by: Michael Pankov <work@michaelpankov.com>

### CODE REVIEW

File: iml-agent/src/cli.rs [hunk 1]
```diff
@@ -99,6 +99,20 @@ pub enum PackageCommand {
     },
 }
 
+#[derive(Debug, StructOpt)]
+pub enum KernelModuleCommand {
```
Comment (jgrund) [inline iml-agent/src/cli.rs]: Can you add some comments so structopt will print the help text for each command?

Comment (mkpankov) [reply iml-agent/src/cli.rs]: Done

File: iml-agent/src/cli.rs [hunk 2]
```diff
@@ -99,6 +99,22 @@ pub enum PackageCommand {
...
+    #[structopt(name = "loaded")]
+    /// Is the module loaded?
+    Loaded {
+        #[structopt(name = "module")]
+        module: String,
+    },
+    #[structopt(name = "version")]
+    /// What is the version of the module?
```
Comment (ip1981) [inline iml-agent/src/cli.rs]: Nit:

```suggestion
    /// Get the version of the module.
```

File: iml-agent/src/cli.rs [hunk 3]
```diff
@@ -99,6 +99,22 @@ pub enum PackageCommand {
     },
 }
 
+#[derive(Debug, StructOpt)]
+pub enum KernelModuleCommand {
+    #[structopt(name = "loaded")]
+    /// Is the module loaded?
```
Comment (ip1981) [inline iml-agent/src/cli.rs]: Nit:
```suggestion
    /// Check if the module is loaded.
```

File: iml-agent/src/cli.rs [hunk 4]
```diff
@@ -242,7 +258,11 @@ pub enum App {
     },
 
     #[structopt(name = "kernel_module")]
-    KernelModule { module: String },
+    /// Get kernel module state and version
```
Comment (ip1981) [inline iml-agent/src/cli.rs]: Nit:
```suggestion
    /// Get information about a kernel module
```

**Pull request merged**